tests/resources/example_project/MLproject

```diff
@@ -20,6 +20,12 @@ entry_points:
     parameters:
       duration: int
     command: "sleep {duration}"
+  greeter_compute_params:
```

Comment:
Could we just add `args` to the `greeter` entry point and not have this separate entry point?

Reply:
Fair point, moved to greeter